src/js/post-911-gib-status/containers/Post911GIBStatusApp.jsx

```diff
@@ -56,7 +57,8 @@ class Post911GIBStatusApp extends React.Component {
                   print a copy of this screen for benefit and eligibility verification.
                 </p>
               </div>
-              <UserInfoSection userData={enrollmentState}/>
+              <UserInfoSection userData={enrollmentData}/>
+              <EnrollmentHistory enrollmentData={enrollmentData}/>
```

Comment:
Yeah, was thinking we should unify these names to make maintenance easier. I think we can do that in one of the next PRs.